src/lib/tableland-calls.ts

```diff
@@ -199,6 +206,8 @@ async function setController(
   return camelCaseKeys(json.result.tx);
 }
 
+// NOTE: tablelandCalls.lockController is not being used because the
+//       Validator RPC endpoint has not been implemented yet.
```

Comment:
I suspect it won't be created either.

Reply:
Should we remove this function altogether? If relaying was to happen putting this back would be simple

Reply:
Let's hold off on any changes here until the discussion around APIs settles.